src/components/app.js

```diff
@@ -35,7 +50,4 @@ class App extends Component {
   }
 }
 
-export default connect(
-  null,
-  null
-)(App)
+export default connect(null, null)(App)
```

Comment:
if it is null,null then why do you connect
just export as App

Reply:
Yea .. it was like that from the previous commit. I have not done that intentionally ... Will change it !